packages/contracts/src/yield/LiquidContinuousMultiTokenVault.sol

```diff
@@ -65,6 +67,7 @@ contract LiquidContinuousMultiTokenVault is
         TripleRateContext(
             params.fullRateScaled,
             params.reducedRateScaled,
+            currentPeriod(),
```

Comment:
best to be explicit, what are we expecting, always 0 ?

we haven't even set the startTime on the internal clock, so calling currentPeriod() seems premature.

Reply:
As the `Timer` is ahead of the `TripleRateContext` in the constructor chain, I thought it works. It doesn't fail at least, but that could be because everything is 0 based.

Yes, I expect this value to always be 0, and I had that value in there, but thought that `currentPeriod()` was clearer.
I will re-assess, confirming that the constructor invocation works or not and setting an appropriate value.

Reply:
I have checked this. The `Timer` is ahead of `TripleRateContext` in the **is** inheritance chain. So, the constructor is invoked first. Thus, we can use `currentPeriod()` as is, and the initial Effective From Period can remain configurable. If we hard-code it to 0, then we lose that.